src/mount/mount-tool.c

```diff
@@ -296,6 +296,31 @@ static int parse_argv(int argc, char *argv[]) {
                         log_error("Listing devices only supported locally.");
                         return -EOPNOTSUPP;
                 }
+        } else if (arg_action == ACTION_UMOUNT) {
+                if (optind >= argc) {
+                        log_error("At least one argument required.");
+                        return -EINVAL;
+                }
+
+                if (argc > optind+1) {
+                        log_error("At most one argument required.");
+                        return -EINVAL;
+                }
+
+                if (is_device_path(argv[optind])) {
```

Comment:
I think you want to move this check to `stop_mount`. Then there'll be no problem with having multiple arguments, you can convert each argument just before using it.